#goal
MakePhotoCommand make_photo_command
---
#result
bool result
---
#feedback
